refactor: LEAP-1290: remove feature flag FF_LOPS_E_10 and related code

Gondragos opened the pull request:
Removed the outdated `fflag_feat_front_lops_e_10_updated_ux_short` feature flag and its associated logic, styles, and assets throughout the codebase. This reverts to the default UI elements where conditional checks for the flag previously existed.

| **Key** | `fflag_feat_front_lops_e_10_updated_ux_short` |
|---|---|
| **Variable Name** | `FF_LOPS_E_10 ` |
| **Ends up as** | 🟥 `False` |
| **Description** | Updated UX with improved accessibility |
| **Reason** | Not relevant anymore |

Gondragos committed:
refactor: remove feature flag FF_LOPS_E_10 and related code

Removed the outdated `fflag_feat_front_lops_e_10_updated_ux_short` feature flag and its associated logic, styles, and assets throughout the codebase. This reverts to the default UI elements where conditional checks for the flag previously existed.

Gondragos commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13900569261)
> Successfully merged:  create mode 100644 web/libs/editor/tests/integration/e2e/control_tags/visibility.cy.ts

Gondragos committed:
Merge branch 'develop' into 'fb-LEAP-1290/stale-ff'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13900569261

hlomzik commented on web/libs/datamanager/src/utils/feature-flags.js:
was this flag marked as stale? because I don't see changes to list of stale FFs

Gondragos replied:
https://github.com/HumanSignal/label-studio/blob/develop/label_studio/core/feature_flags/stale_feature_flags.py#L7
That was done not in that PR.

Gondragos replied:
oh... you mean removing...

Gondragos committed:
fix: remove stale feature flag from list

Removed the feature flag 'fflag_feat_front_lops_e_10_updated_ux_short' as it is no longer in use.
